libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationContract.java

```diff
@@ -41,6 +41,8 @@
 
     void updateNavigationMap(Location location);
 
+    void updateWayNameEnabled(boolean isEnabled);
```

Comment:
Can we disable without adding a new `NavigationContract` API? Maybe adding a check here https://github.com/mapbox/mapbox-navigation-android/blob/50722592109c86657c8c746ba142a74cfb87ea85/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/MapWayName.java#L39?